README.md

```diff
@@ -1,6 +1,18 @@
-# Deploy ELK on Kubernetes in Azure Container Service (ACS)
+# Deploy Elastic Stack on Kubernetes in Azure Container Service (ACS)
 
-This repository contains tools and helm charts to help deploy the [ELK stack](https://www.elastic.co/products) on [Kubernetes](https://kubernetes.io/) in [Azure Container Service (ACS)](https://docs.microsoft.com/azure/container-service/).
+This repository contains tools and helm charts to help deploy the [Elastck stack](https://www.elastic.co/products) on [Kubernetes](https://kubernetes.io/) in [Azure Container Service (ACS)](https://docs.microsoft.com/azure/container-service/).
+
+## How the solution works
+
+* Deploy a Kubernetes cluster on Azure.
+* Deploy a Virtual Machine served as the Controller Node to manage and configure Kubernetes cluster on Azure.
+* Register Controller Node's FQDN as the entry to Kubernetes dashbord.
+* Authentication supported for Kubernetes dashbord:
+    * Username / Password
+    * [Azure Active Directory OAuth 2.0](https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-protocols-oauth-code)
+* Deploy a Azure Container Registry if no public registry is provided.
+* Build docker images for Elastic Stack and push images to user specified public registry or the Azure Container Register.
```

Comment:
The script doesn't push anything to the public registry. Assume it can pull images from the given public registry: https://github.com/Microsoft/elk-acs-kubernetes/blob/logstash-eventhub-aztable/scripts/run.sh#L221